App.js

```diff
@@ -52,44 +55,57 @@ const HomeScreen = ({ navigation }) => {
-                        <HomeButton
-                            text="PLAY GAME"
-                            onPress={() => navigation.navigate('Play')}
-                            source={require('./assets/genconnect_ombre_allaboutme.png')}
-                        />
-                        <HomeButton
-                            text="HOW TO PLAY"
-                            onPress={() => navigation.navigate('How To Play')}
-                            source={require('./assets/genconnect_ombre_brightfuture.png')}
-                        />
-                        <HomeButton
-                            text="PARENT GUIDE"
-                            onPress={() => navigation.navigate('Parent Guide')}
-                            source={require('./assets/genconnect_ombre_whatwouldyoudo.png')}
-                        />
-                        <HomeButton
-                            text="PARENT TIPS"
-                            onPress={() => navigation.navigate('Parent Tips')}
-                            source={require('./assets/genconnect_ombre_dancechallenge.png')}
-                        />
-                    </StyledHomeButtonView>
-                    <StatusBar style="auto" />
-                </ScrollView>
+                <StyledHomeButtonView>
+                    <HomeButton
+                        text="PLAY GAME"
+                        onPress={() => navigation.navigate('Play')}
+                        source={require('./assets/genconnect_ombre_allaboutme.png')}
+                    />
+                    <HomeButton
+                        text="HOW TO PLAY"
+                        onPress={() => navigation.navigate('How To Play')}
+                        source={require('./assets/genconnect_ombre_brightfuture.png')}
+                    />
+                    <HomeButton
+                        text="PARENT GUIDE"
+                        onPress={() => navigation.navigate('Parent Guide')}
+                        source={require('./assets/genconnect_ombre_whatwouldyoudo.png')}
+                    />
+                    <HomeButton
+                        text="PARENT TIPS"
+                        onPress={() => navigation.navigate('Parent Tips')}
+                        source={require('./assets/genconnect_ombre_dancechallenge.png')}
+                    />
+                </StyledHomeButtonView>
+                <StatusBar style="auto" />
+                <TouchableOpacity
+                    onPress={() => navigation.navigate('Information')}
+                    style={{
+                        position: 'absolute',
+                        right: 20,
+                        top: 40,
+                    }} >
+                    <Image
+                        source={require('./assets/footer_info.png')}
+                        style={{
+                            width: 50,
+                            height: 50
+                        }}
```

Comment:
and here

Reply:
but yea can just leave it like this since we are close to the deadline. should avoid this in the future